packages/desktop-client/src/browser-preload.browser.js

```diff
@@ -1,6 +1,6 @@
 import { initBackend as initSQLBackend } from 'absurd-sql/dist/indexeddb-main-thread';
-// eslint-disable-next-line import/no-webpack-loader-syntax
-import BackendWorker from 'worker-loader!./browser-server';
+
+const BackendWorker = new URL('./browser-server.js', import.meta.url);
```

Comment:
Since this isn’t a class/component:

```suggestion
const backendWorkerURL = new URL('./browser-server.js', import.meta.url);
```

Reply:
Makes sense: https://github.com/actualbudget/actual/pull/697/commits/710da91267c26f8275ce8fe6ebf7e5389e7cbb1c